### ISSUE TITLE
Send Poll for Likelihood of Referral

### ISSUE THREAD

**Issue by loripusey** [CDCgov/prime-reportstream#3690]
## Problem Statement
As the Experience Team, we would like to poll our current users so that we can find out the likelihood that they would recommend using ReportStream to their peers.

## Acceptance Criteria
- Send poll to all STLTS and reporting entities currently in production on ReportStream (exclude Simple Report)

**Comment by victorialb-usds** [comment]
VLB to get touchpoints account. Existing questions PRA approved.

**Comment by victorialb-usds** [comment]
@Adrian-Brewster and @BerniXiongA6 Given I will be out of the office next week there is a potential that you all might be able to send the survey out via RS inbox. I will be back 4/11 so if you want to wait I would rather make sure all the i's are dotted  and we do it well. here are some steps I have outlines for what needs to happen based on our previous discussions around the directory. 

- [ ] (1) Draft email - @BerniXiongA6 will add VLB signature and @Adrian-Brewster if there are any questions coming in you can forward to me? (I want to note the labor there -- is that ok?). Instead of putting their name and doing one by one. We could say Dear Valued ReportStream user, Or delete the salutation completely. 
- [ ] (2) test it by sending it to Berni/Rachel to make sure you can open it etc.
- [ ] (3) once we get the OK from Berni and Rachel (lets say you two but could be others. 
- [ ] (4) Adrian can send it with all the highlighted emails BCC'd 
- [ ] AB monitor inbox for any emails that bounce back and forward to Victoria, Berni and Rachel 
- [ ] BX log any bounces 
- [ ] AB updates the directory and decides on who to send to as alternate
- [ ] Send to another contact at the organization. 
Note: I am ok with multiple people responding per organization as we will aggregate by organization, in case people want to forward it on that will be fine.

**Comment by victorialb-usds** [comment]
Depending on what the response rate it we can send a reminder email --- access to survey is for USDS only @rachelhanster or @drew-usds if you get access you can see the survey while I am gone if needed.

**Comment by BerniXiongA6** [comment]
@Adrian-Brewster Happy to work with you on this async and we can even sync up if needed too! I'll look at what I need to prepare for this work. Thanks for the detailed info @victorialb-usds

**Comment by Adrian-Brewster** [comment]
Looks good to me @victorialb-usds

**Comment by BerniXiongA6** [comment]
@Adrian-Brewster It's OK to now copy/paste [the following email verbiage](https://cdc.sharepoint.com/:w:/t/USDSatCDC/ETeTRzz8QrdFjmX2IFADT2IB5tKNklJsjmyVlA-cb9XPHg?e=B2Kjd8) into your RS Outlook email draft. I've wordsmithed it slightly to add a due date (assuming we send it out to users on Mon or Tues next week). Also, I added "Action Requested:" in the subject line because it alerts the recipient that they need to do something with the email. And I've hyperlinked the correct survey link. Want to do a test run by sending the email to @rachelhanster @HeatherRienks @victorialb-usds and @BerniXiongA6 (me) on Friday 4/1/2022?
